src/main/scala/org/labrad/registry/Registry.scala

```diff
@@ -69,6 +68,7 @@ object Registry {
 }
 
 class Registry(val id: Long, val name: String, store: RegistryStore, externalConfig: ServerConfig)
+              (implicit ec: ExecutionContext)
```

Comment:
I like this style of formatting. I recognize that this is a broader discussion, but how do you feel about aligning all class stuff to the first parens, e.g.
```
class Registry(val id: Long, val name: String, store: RegistryStore, externalConfig: ServerConfig)
              (implicit ec: ExecutionContext)
              extends LocalServer with Logging {
```

It's not quite [what the Scala community suggests](http://docs.scala-lang.org/style/declarations.html#classes), but it would be visually easier to parse.